babel/messages/extract.py

```diff
@@ -835,10 +835,12 @@ def parse_template_string(
         if level:
             expression_contents += character
         if not inside_str:
-            if character == '{' and prev_character == '$':
+            # start of new template string or object definition
+            if character == '{':
```

Comment:
How would this handle something like
```javascript
hello(`there is {nothing} to worry about`)
```
?
AISI, `nothing` would then be passed to `extract_javascript`?